_posts/2019-11-01-snowplow-r117-biskupin/index.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title-short: "Snowplow R117 release"
+title: "Snowplow R117 release"
+tags: [snowplow, release]
+author: Łukasz Indykiewicz
+category: Releases
+permalink: /blog/2019/11/01/snowplow-r117-biskupin/
```

Comment:
The date (here and in filename)

Reply:
Set to 12.02 - let's hope this is the date :)